Mapc2 Deliverable 0

nathanlauer opened the pull request:
This pull request contains three primary things:
- initial project description from pitch day
- notes from the kickoff meeting
- Deliverable 0: project proposal

nathanlauer committed:
Added a .gitignore file

gallettilance commented on .gitignore:
```diff
@@ -0,0 +1 @@
+.DS_Store
```
great! thank you for adding this

mjadick committed:
Opioid Crisis Project uploading weekly SCRUM report-02/26/21

abhisheksamal committed:
added Scrum-1 file for DDM4RPD_T1

gian-s committed:
Adding Notebook for Deliverable 1

nathanlauer committed:
Simple script which demonstrates how to get ookla data from aws s3

nathanlauer committed:
Merge branch 'mapc2' of github.com:nathanlauer/CS506Spring2021Repository into mapc2

tmarchBU committed:
deliverable 1 completed

astreich4 committed:
added notes

astreich4 committed:
Merge remote-tracking branch 'origin/mapc2' into mapc2

nathanlauer committed:
Created folder client_meetings, moved relevant files there

nathanlauer committed:
Updated script to pull Ookla data for each quarter of the passed in year

nathanlauer committed:
Removed old test script

nathanlauer committed:
Updated gitignore to not track parquet files

nathanlauer committed:
Updated gitignore with standard config for python

Commit:
Project Deliverable 1

njmosca committed:
editing and adding path_mover.py

eviewan committed:
adding financial metrics parsing file

rachelvpeng committed:
deliverable1 completed

eviewan committed:
adding scrum file

e-south committed:
Merge pull request #2 from e-south/data_gathering

Data gathering

e-south committed:
Merge pull request #3 from ewan212/master

adding financial metrics parsing file

nomatu committed:
Add part 1 of images

nomatu committed:
For Deliverable 1

Part 2, 3, 4

g-asokan committed:
Merge pull request #28 from yashdjain/master

Submitting Project Deliverable 0 & 1

nomatu committed:
Create LatinX_Republican_Support_deliverable_1.txt

Deliverable 1

e-south committed:
adding deliverable 1 and gitignore

e-south committed:
Delete .ipynb_checkpoints directory

e-south committed:
Delete .DS_Store

nomatu committed:
Rename LatinX_Republican_Support_deliverable_1.txt to LatinX_Republican_Support_deliverable_1.md

nomatu committed:
Merge pull request #1 from nomatu/new_branch

Rename LatinX_Republican_Support_deliverable_1.txt to LatinX_Republic…

nomatu committed:
Delete Deliverable_0.md

nomatu committed:
Merge pull request #2 from nomatu/new_branch

Delete Deliverable_0.md

nomatu committed:
Add files via upload

nomatu committed:
Add files via upload

gian-s committed:
Adding code in notebook file for Deliverable 1

relishyeah committed:
Added deliverable 1

cody9898 committed:
added my python notebook with some basic visualizations

cody9898 committed:
added graphs with children in 3 aggression categories

mjadick committed:
Opioid Crisis Project: Uploading Deliverable 1 and gofundme_scrape.py, a preliminary tool to scrape urls from GoFundMe

ryan-shnitman committed:
added deliverable 1 (combine my overview doc and Cody's ipynb)

ryan-shnitman committed:
added last page to deliverable 1

ryan-shnitman committed:
final deliverable 1

pengyou200902 committed:
add gitignore

cody9898 committed:
Merge pull request #3 from ryan-shnitman/master

final deliverable 1

pengyou200902 committed:
nest proposal with team name

abhisheksamal committed:
added Deliverable_1 files for DDM4RPD_T1

pengyou200902 committed:
add deliverable 1

abhisheksamal committed:
Datafile name correction in python files

gallettilance committed:
Merge pull request #10 from tmarchBU/master

Legalization and Drug Deaths

jfeng5529 committed:
Merge pull request #47 from cody9898/master

Drift Diffusion Models of children's interactions in a Repeated Prisoner's Dilemma Game (Team 2) Deliverable 1

jfeng5529 committed:
Merge pull request #32 from abhisheksamal/master

Added Scrum_1 report and Deliverable_1 files of Team_1 for Drift Diffusion Models of children's interactions in a Repeated Prisoner's Dilemma Game.

jfeng5529 committed:
Merge pull request #31 from mjadick/master

Opioid Crisis Project uploading weekly SCRUM report-02/26/21

ktllee committed:
add: project folder, description, deliverable0 folder

ktllee committed:
add: README.md and pdf versions of revised proposal

szhen115 committed:
Merge pull request #1 from ktllee/master

files for deliverable0

ZhixingZhao committed:
Upload the code to get data for MA and convert to csv

khaledsulayman committed:
add Khaled github uname

e-south committed:
Delete .DS_Store

e-south committed:
Delete .DS_Store

e-south committed:
Delete .DS_Store

szhen115 committed:
Merge pull request #2 from khaledsulayman/master

add Khaled github uname

sgorok committed:
Merge pull request #2 from sgorok/deliverable1

Deliverable1 merged with Master

sgorok committed:
merging master and other branches to fix issue with README

sgorok committed:
merging master and other branches to fix issue with README (and deleting extra files)

ZhixingZhao committed:
Fix Some Bugs

The getting data through a function doesn't seems to work well. Might change to a big for loop.

ZhixingZhao committed:
Pushed New Version

ZhixingZhao committed:
FInal Version

Just run las three blocks should be able to get the data related to MA

szhen115 committed:
Update README.md

updated deliverable 0 status

nathanlauer committed:
brief notes from ad hoc meeting with Steve

nathanlauer committed:
Added folder for deliverables, and draft for deliverable 1

nathanlauer committed:
Wrote example script for bigquery cli access, and python script for iterating over days to run big query

nathanlauer committed:
a few typo fixes

nathanlauer committed:
Adding the csv files for Ookla data

nathanlauer committed:
Removed ignore csv

nathanlauer committed:
Added the final quarter for ookla data

nathanlauer committed:
Added csv for 8am to 830am mlab data

gallettilance committed:
Updating: incorrect file name

szhen115 committed:
Merge branch 'master' of https://github.com/BU-Spark/CS506Spring2021Repository

nathanlauer committed:
corrected typo in script name

rishabnayak committed:
Update README.md

rachelvpeng committed:
updated scraper to include allegations and finding fields

rishabnayak committed:
Merge pull request #56 from BU-Spark/test

Update README.md

rishabnayak committed:
Merge pull request #30 from lingxiaoyuan/master

Upload the second Scrum report

rishabnayak committed:
Update README.md

rishabnayak committed:
Update README.md

rishabnayak committed:
Merge pull request #57 from BU-Spark/test

update readme

ankilinnn committed:
Merge branch 'master' into master

ankilinnn committed:
Merge pull request #3 from pengyou200902/master

Change deliverable 0 & new deliverable 1

nathanlauer committed:
Moved script mlab by time to date, for easier processing

nathanlauer committed:
2020 mlab data for 8-8:30, 12-12:30, 3:30-4:00, 8pm-8:30pm

nathanlauer committed:
removed old mlab csv file

nathanlauer committed:
Small update to deliverable 1 report, added data size section for MLAB

rachelvpeng committed:
fixed bug

nathanlauer committed:
Final update to deliverable 1 report, added checklist in summary section

nathanlauer committed:
fixed merge conflict

nathanlauer committed:
Merge pull request #1 from nathanlauer/mapc2

Mapc2

e-south committed:
updating html_parser and adding topic models

njmosca committed:
Merge pull request #4 from e-south/topic_models

updating html_parser and adding topic models

sgorok committed:
merged deliverable1 to master

jenajjedu committed:
Added Deliverable 1

XiongxinZ committed:
Give a general view of our data and remove some "Unclassified"

jenajjedu committed:
Added Scrum 1 Report

kimonazj committed:
create deliverable 1 folder

kimonazj committed:
add deliverable 1

1. result_D1
2. scrum
3. pulled datasets

kimonazj committed:
Delete Deliverable 1 directory

kimonazj committed:
Create test

kimonazj committed:
add Deliverable 1

1. result_D1
2. scrum
3. pulled datasets

kimonazj committed:
Delete test

XiongxinZ committed:
Calculate the waste volume for top coffee categories

We found that the top 20 coffee products requested are K-Cup Pods and Coffee Creamers and then calculated the waste volume based on the online open source of data for K-Cup and Coffee Creamer.

jennyli1299 committed:
Notes for second client meeting

jennyli1299 committed:
Notes for second client meeting

jennyli1299 committed:
Update to notes for second client meeting

jennyli1299 committed:
Update to notes for second client meeting

mjadick committed:
Opioid Crisis Project-- uploading SCRUM Report 2 03/05/21

Commit:
set up the structure for all deliverables

abhisheksamal committed:
added Scrum-2 file for DDM4RPD_T1

Commit:
Update details for Deliverable 1

szhen115 committed:
structured repo by code vs data vs deliverables

szhen115 committed:
created folders

szhen115 committed:
Merge branch 'master' of https://github.com/BU-Spark/CS506Spring2021Repository

szhen115 committed:
Merge branch 'master' into folder-restructure

szhen115 committed:
added folders again

ktllee committed:
add: preserved deliverable descriptions

sgorok committed:
change in directory

sgorok committed:
set remote upstream & fetched upstream master

g-asokan committed:
Merge pull request #66 from sgorok/master

Most Recent Updates (Police Conduct)

richardjlee6 committed:
Tested other pdf parsers

szhen115 committed:
filtering out relevant cols

szhen115 committed:
pulled deliverables desc

szhen115 committed:
Merge branch 'master' into preprocess

ankilinnn committed:
Merge pull request #8 from milesway/master

Deliverable 0 & 1 - BU Sustainable Purchasing Program

szhen115 committed:
sanitized data

ankilinnn committed:
Merge pull request #7 from kimonazj/master

Deliverable 0 and 1 Pull Request

rishabnayak committed:
Create CONTRIBUTING.md

rishabnayak committed:
Update CONTRIBUTING.md

relishyeah committed:
Created Deliverable 2 File

xav-ie committed:
added React skeleton

xav-ie committed:
Merge branch 'master' of https://github.com/relishyeah/CS506Spring2021Repository

gian-s committed:
updating SCRUM files

Commit:
with deliverable 1

relishyeah committed:
Create 311_census_combo.ipynb

relishyeah committed:
Update .gitignore

szhen115 committed:
Merge pull request #4 from Zayta/preprocess

Preprocess

ktllee committed:
update: removed original datasets from remote, shuffled files accordingly

dataset_ignore has no datasets in GitHub, just README.md
datasets_clean has clean dataframes as csv
appropriate paths changed

relishyeah committed:
Update 311_census_combo.ipynb

liudarren3 committed:
Adding the MAPC data

Fixing error: adding back the other project folders

szhen115 committed:
updating open space data

added site name and ownership back to open space dataset

sorted open space by size

szhen115 committed:
added dataset_ignore to gitignore file

ktllee committed:
add: files for cleaning master businesses dataset

ktllee committed:
add: files for deliverable1 report

ktllee committed:
add: directory for SCRUM files

Commit:
update d1

jennyli1299 committed:
3_09_21 Meeting Notes

jennyli1299 committed:
3_09_21 Meeting Notes

gallettilance committed:
Merge pull request #42 from e-south/master

10K_Risk_Factors Deliverable 1

xav-ie committed:
Merge branch 'master' into master

kamranarif committed:
Merge pull request #70 from Zayta/deliverable1

Deliverable1

kamranarif committed:
Merge pull request #58 from nathanlauer/master

MAPC Team 2 Deliverable 1

jenajjedu committed:
The first scrum report got deleted.. added it back.

jenajjedu committed:
renamed the scrum reports to remove spaces in the name

meganmastro committed:
created Civera folder, added deliverables 0 and 1

meganmastro committed:
added revised project SOW as pdf

kamranarif committed:
Merge pull request #17 from mcxxmc/master

deliverable 0 and 1 for Boston 15-minute city Team 1

sunkcostsong committed:
Merge pull request #1 from BU-Spark/master

pulled from course project repo

sunkcostsong committed:
Create temp

sunkcostsong committed:
Rename LatinX Republican Support/LatinX_Republican_Support_Deliverable_0.md to LatinX Republican Support/Deliverables/LatinX_Republican_Support_Deliverable_0.md

sunkcostsong committed:
Delete temp

sunkcostsong committed:
Create Scrum

sunkcostsong committed:
Delete Scrum

sunkcostsong committed:
Create temp

sunkcostsong committed:
Added missing scrum notes

sunkcostsong committed:
Delete temp

sunkcostsong committed:
Merge pull request #2 from SunkCostSong/Draft

Reorganized and added scrum notes

jfeng5529 committed:
Merge pull request #62 from mjadick/master

Opioid Crisis Project-- uploading SCRUM Report 2 03/05/21

jfeng5529 committed:
Merge pull request #64 from abhisheksamal/master

Added Scrum-2 file for Drift Diffusion Models of children's interactions in a Repeated Prisoner's Dilemma Game Team_1

rishabnayak committed:
Merge pull request #72 from meganmastro/civera

abhisheksamal committed:
added Scrum-3 file for DDM4RPD_T1

abhisheksamal committed:
added Scrum-3 file for DDM4RPD_T1

Commit:
add weekly SCRUM file

ryan-shnitman committed:
SCRUM-2, changing file names, analysis files

ryan-shnitman committed:
scrum folder

ryan-shnitman committed:
removing analysis files

mjadick committed:
Opioid Crisis Project -- uploading SCRUM report 3 3.12.21

jfeng5529 committed:
Merge pull request #76 from abhisheksamal/master

Added Scrum-3 for Drift Diffusion Models of children's interactions in a Repeated Prisoner's Dilemma Game Team_1

jfeng5529 committed:
Merge pull request #79 from mjadick/master

Opioid Crisis Project -- uploading SCRUM report 3 3.12.21

lingyanj committed:
Merge pull request #43 from nomatu/master

LatinX_Republican_Support_Deliverable_1

jfeng5529 committed:
Merge pull request #78 from ryan-shnitman/master

adding scrum, updating file names, removing analysis

danieldelijani committed:
Created new csv files with preprocessed names

nathanlauer committed:
Added map of AS number to ASName and description

nathanlauer committed:
Added map of AS number to ASName and description

nathanlauer committed:
Added a small dataset for MLAB, with just 10000 rows, for testing purposes

nathanlauer committed:
Added a small dataset for MLAB, with just 10000 rows, for testing purposes

nathanlauer committed:
Some cleanup in the as_map.csv file

nathanlauer committed:
Some cleanup in the as_map.csv file

nathanlauer committed:
Added script to add ProviderName to MLAB data

nathanlauer committed:
Added script to add ProviderName to MLAB data

nathanlauer committed:
Added new MLAB data file with provider name

nathanlauer committed:
Added new MLAB data file with provider name

nathanlauer committed:
Added provider name to small mlab data file

nathanlauer committed:
Added provider name to small mlab data file

mjadick committed:
Opioid Crisis Project -- uploading SCRUM report 4 3.18.21

jennyli1299 committed:
grouped data by city in csv

jennyli1299 committed:
grouped data by city in csv

jennyli1299 committed:
Undo adding unnecessary csv

jennyli1299 committed:
Added MAPC municipalities csv

jennyli1299 committed:
csv: specific MAPC cities

jennyli1299 committed:
csv: specific MAPC cities

jennyli1299 committed:
py file to grab data by city

jennyli1299 committed:
py file to grab data by city

astreich4 committed:
adding script to get mlab avg per city per provider

astreich4 committed:
adding script to get mlab avg per city per provider

lingyanj committed:
Merge pull request #59 from jenajjedu/delv_1

WGBH-DCAMM, Deliverable 1, cleaned up PR

lingyanj committed:
Merge pull request #60 from jenajjedu/scrum_1

WGBH-DCAMM, Scrum 1

jennyli1299 committed:
Notes from client meeting on 3.19.21

jennyli1299 committed:
Notes from client meeting on 3.19.21

jennyli1299 committed:
mv csv locations for MAPC region

jennyli1299 committed:
mv csv locations for MAPC region

sgorok committed:
merged deliverable 2 with master

ZhixingZhao committed:
Add data/ to gitignore and group Ookla data by cities

ZhixingZhao committed:
Add data/ to gitignore and group Ookla data by cities

lingyanj committed:
Merge pull request #45 from relishyeah/master

City of Revere Service Data Project: Deliverable 1

lingyanj committed:
Merge pull request #74 from SunkCostSong/master

Reorganized team project repo

lingyanj committed:
Merge pull request #77 from mcxxmc/master

add weekly SCRUM file

sgorok committed:
removed files unrelated

g-asokan committed:
Merge pull request #84 from sgorok/master

Merge for D2 (not finalized but needs review)

nathanlauer committed:
Added my notes from client meeting on 3_19

nathanlauer committed:
Added my notes from client meeting on 3_19

nathanlauer committed:
Started writing deliverable2

nathanlauer committed:
Started writing deliverable2

nathanlauer committed:
Added index=False to add_as_name_to_mlab.py script, then ran it on mlab data, and updated other scripts to use that one instead of the unnecessary other one

nathanlauer committed:
Added index=False to add_as_name_to_mlab.py script, then ran it on mlab data, and updated other scripts to use that one instead of the unnecessary other one

nathanlauer committed:
Removed Unnamed: 0 column, which had previously referenced index column

nathanlauer committed:
Removed Unnamed: 0 column, which had previously referenced index column

nathanlauer committed:
Deleted unnecessary extra mlab file

nathanlauer committed:
Deleted unnecessary extra mlab file

nathanlauer committed:
Added new file with a list of Providers found in MAPC work for their three cities. Had to put it in a different folder tho, for some reason git wouldn't let me add it to the data folder

nathanlauer committed:
Added new file with a list of Providers found in MAPC work for their three cities. Had to put it in a different folder tho, for some reason git wouldn't let me add it to the data folder

nathanlauer committed:
Added a script to extract full list of MLAB providers, and the generated list

nathanlauer committed:
Added a script to extract full list of MLAB providers, and the generated list

nathanlauer committed:
Added script to compute descriptive stats on the MLAB data

nathanlauer committed:
Added script to compute descriptive stats on the MLAB data

nathanlauer committed:
File produced from mlab stats script

nathanlauer committed:
File produced from mlab stats script

nathanlauer committed:
Updated draft of deliverable 2

jfeng5529 committed:
Merge pull request #81 from mjadick/master

Opioid Crisis Project -- uploading SCRUM report 4 3.18.21

nathanlauer committed:
Further updates to deliverable 2

nathanlauer committed:
Further updates to deliverable 2

nathanlauer committed:
Udpated draft for deliverable 2

nathanlauer committed:
Udpated draft for deliverable 2

nathanlauer committed:
Merge branch 'mapc2' of github.com:nathanlauer/CS506Spring2021Repository into mapc2
